gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -4088,6 +4081,15 @@ gnc_plugin_page_register_cmd_reverse_transaction (GtkAction* action,
         return;
     }
 
+    new_trans = xaccTransGetReversedBy (trans);
+    if (new_trans)
+    {
+        if (!gnc_verify_dialog (GTK_WINDOW (window), TRUE, "%s",
+                               _("A reversing entry has already been created for this transaction. Jump to transaction?")))
```

Comment:
I wonder if we should split the declarative sentence and the question in different lines, perhaps separated by an empty line.
From my expirience that could be faster understood.

Reply:
Splitting done